File: 프로그래머스/lv1/64061. 크레인 인형뽑기 게임/크레인 인형뽑기 게임.py
def solution(board, moves):
    basket = []
    count = 0
    for i in moves:
        for j in range(len(board)):
            if board[j][i-1] != 0:
                if len(basket) > 0 and basket[len(basket)-1] == board[j][i-1]:
                    basket.pop(len(basket)-1)
                    count += 2
                    board[j][i-1] = 0
                    break
                else:
                    basket.append(board[j][i-1])
                    board[j][i-1] = 0
                    break
    print(basket)
    return count

File: 프로그래머스/lv1/64061. 크레인 인형뽑기 게임/크레인 인형뽑기 게임.js
function solution(board, moves) {
    const basket = [];
    let answer = 0;
    for(move of moves){
        for(let i=0; i < board.length; i++){
            const pick = board[i][move - 1]
            if(pick !== 0) {
                basket.push(pick)
                board[i][move - 1] = 0
                
                if(basket.at(-1) === basket.at(-2)){
                    basket.pop();
                    basket.pop();
                    answer+=2
                }
                break
            }
        }
    }
    return answer
}